mmv1/products/spanner/terraform.yaml

```
@@ -139,6 +139,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       post_create: templates/terraform/post_create/sleep.go.erb
       pre_delete: 'templates/terraform/pre_delete/spanner_instance.go.erb'
       constants: 'templates/terraform/constants/spanner_instance.go.erb'
+      pre_update: 'templates/terraform/pre_update/spanner_force_destroy.go.erb'
```

It would be great to keep the name consistent in case other things need to be added to this file later.

```suggestion
      pre_update: 'templates/terraform/pre_update/spanner_instance.go.erb'
```